### ISSUE TITLE
Add support for #EXT-X-SESSION-KEY

### ISSUE THREAD

**Issue by stoyanovgeorge** [globocom/m3u8#257]
Could you please add support to `#EXT-X-SESSION-KEY` (https://datatracker.ietf.org/doc/html/rfc8216#section-4.3.4.5). 

Example Manifest: 
```
#EXTM3U
#EXT-X-VERSION:6
#EXT-X-INDEPENDENT-SEGMENTS
#EXT-X-SESSION-KEY:METHOD=SAMPLE-AES,URI="skd://c8d3aa78-e8b7-723c-d51e-b6e303b4bb4d:34ABD5E1D7311F058E44B4A1B585B825",KEYFORMAT="com.apple.streamingkeydelivery",KEYFORMATVERSIONS="1"
#EXT-X-MEDIA:TYPE=AUDIO,GROUP-ID="audio1",NAME="SPA",DEFAULT=YES,AUTOSELECT=YES,LANGUAGE="SPA",URI="06.m3u8"
#EXT-X-MEDIA:TYPE=SUBTITLES,GROUP-ID="subs",NAME="caption_1",DEFAULT=NO,AUTOSELECT=YES,LANGUAGE="SPA",URI="107.m3u8"
#EXT-X-MEDIA:TYPE=SUBTITLES,GROUP-ID="subs",NAME="caption_2",DEFAULT=NO,AUTOSELECT=NO,LANGUAGE="ENG",URI="108.m3u8"
#EXT-X-MEDIA:TYPE=SUBTITLES,GROUP-ID="subs",NAME="caption_3",DEFAULT=NO,AUTOSELECT=NO,LANGUAGE="ENG",URI="109.m3u8"
#EXT-X-MEDIA:TYPE=SUBTITLES,GROUP-ID="subs",NAME="caption_4",DEFAULT=NO,AUTOSELECT=NO,LANGUAGE="ENG",URI="110.m3u8"
#EXT-X-MEDIA:TYPE=CLOSED-CAPTIONS,GROUP-ID="cc",NAME="caption_1",DEFAULT=NO,AUTOSELECT=YES,LANGUAGE="SPA",INSTREAM-ID="CC1"
#EXT-X-MEDIA:TYPE=CLOSED-CAPTIONS,GROUP-ID="cc",NAME="caption_2",DEFAULT=NO,AUTOSELECT=YES,LANGUAGE="ENG",INSTREAM-ID="CC2"
#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=556518,AVERAGE-BANDWIDTH=463765,HDCP-LEVEL=NONE,CODECS="avc1.42801E,mp4a.40.2",RESOLUTION=320x240,FRAME-RATE=29.970,AUDIO="audio1",SUBTITLES="subs",CLOSED-CAPTIONS="cc"

```

**Comment by davemevans** [comment]
Looks like it's already supported (added by https://github.com/globocom/m3u8/pull/172), it just appears to be missing from the list of supported tags.

**Comment by davemevans** [comment]
Fixed via #262.
